### DISCUSSION TITLE
[Kong] [Grpc-Gateway] : is it supported sending map as query param?

### DISCUSSION THREAD

**Discussion by kagura21** [Kong/kong#12253] [Help] [answered]
Hi guys, so i have proto file like this as an example 

```
service  Service {
    rpc Get(Request) returns (Response) {
        option (google.api.http) = {
            get: "/data/v1/path"
        };
    };
}

message Request {
    map<string, string> query_params = 1; 
}
```

I tried sending the param like this based on what i saw on grpc-gateway github:
`/data/v1/path?query_params[test]=abc`

but when I checked the backend side the request I received was empty. So want to make sure if i sending the correct format? or Map as query params isn't supported by kong?

Thank you in advance

**Comment by StarlightIbuki** [comment, marked as answer]
You should send the request in a form like below:

```
/data/v1/path?query_params.test=abc
```
